libs/frontend/ui-build/src/components/theme-menu/GoogleFontSelect.vue

```diff
@@ -8,9 +8,16 @@
     :options="GoogleFontNames"
     :clearable="false"
     :allowAny="true"
+    :customLabel="true"
     @select="emit('update:modelValue', $event)"
     @click.stop
-  />
+  >
+    <template #default="{ label }">
+      <div :style="{ 'font-family': quotedFont(label) }">
```

Comment:
Hi @sampullman 

In addition to the preview text at the bottom, I also applied the corresponding font to the option label in the list.
But we probably don't want to download all of the fonts all at once just for the option label, so currently only the font that's previously selected will be displayed. I hope this is okay 😬.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/de97eb1d-2a7a-4270-9ce1-8dbbde5743e3

Reply:
I was thinking about the best way to handle this as well. I think your solution is good!